src/pages/UIComponentsPage.js

```diff
@@ -27,6 +30,46 @@ export const UIComponentsPage = () => {
           <SearchPlaceIcon className="icon" />
           <ShareLocationIcon className="icon" />
           <UserLocationIcon className="icon" />
+          <SettingsIcon className="icon" />
+          <HelpIcon className="icon" />
```

Comment:
Would there be a way that all of these icons just have the className icon already? Do they ever _not_ have that className?